Sources/StytchUI/StytchB2BUIClient/Managers/PasswordAuthenticateResetManager.swift

```diff
@@ -0,0 +1,75 @@
+import Foundation
+import StytchCore
+
+struct PasswordAuthenticateResetManager {
+    static func emailEligibleForJITProvisioning(
+        emailAddress: String,
+        emailAllowedDomains: [String]?,
+        emailJitProvisioning: StytchB2BClient.EmailJitProvisioning?
+    ) -> Bool {
+        guard emailJitProvisioning == StytchB2BClient.EmailJitProvisioning.restricted else {
```

Comment:
I'm having a hard time parsing this function, but it seems like this first guard would return false if emailJitProvisioning == 'NOT_ALLOWED', but according to the JS SDK, if it's 'NOT_ALLOWED' it should return true. Which sounds wrong as I'm typing it up lol. Maybe we should sync on this.

Reply:
You are understanding this correctly. Lets discuss.